index.html

```diff
@@ -553,18 +553,14 @@ <h3>
           language can also help with directionality).
         </p>
         <p>
-          A <dfn>language tag</dfn> is a <a>string</a> that matches the
-          production of a `Language-Tag` defined in the [[BCP47]]
-          specifications (see the <a href=
-          "https://www.iana.org/assignments/language-subtag-registry">IANA
-          Language Subtag Registry</a> for an authoritative list of possible
-          values). That is, a language range is composed of one or more
-          <dfn>subtags</dfn> that are delimited by a U+002D HYPHEN-MINUS ("-").
-          For example, the '`en-AU`' language range represents English as
-          spoken in Australia, and '`fr-CA`' represents French as spoken in
-          Canada. Language tags that meet the validity criteria of [[RFC5646]]
-          section 2.2.9 that can be verified without reference to the IANA
-          Language Subtag Registry are considered structurally valid.
+          A <dfn>language tag</dfn> is a [=string=] that matches the production
+          of a well-formed `Language-Tag` defined in [[BCP47]].
```

Comment:
Nit: "well-formed" is slightly redundant here, since it means that the string matches the ABNF in `Language-Tag`, but I think it's good to use the keyword like this.